storybook/qmlTests/tests/tst_SwapInputPanel.qml

```diff
@@ -442,12 +443,11 @@ Item {
 
             numberTestedString = ""
             numberTested = 0
-            mouseClick(amountToSendInput)
             controlUnderTest.tokenAmount = numberTestedString
-            waitForRendering(amountToSendInput)
+            waitForItemPolished(controlUnderTest)
 
-            compare(amountToSendInput.input.text, numberTestedString)
-            compare(controlUnderTest.value, numberTested)
+            tryCompare(amountToSendInput.input, "text", numberTestedString)
+            tryCompare(controlUnderTest, "value", numberTested)
             compare(controlUnderTest.rawValue, AmountsArithmetic.fromNumber(numberTested, modelItemToTest.decimals).toString())
             compare(controlUnderTest.valueValid, false)
             compare(controlUnderTest.selectedHoldingId, tokenKeyToTest)
```

Comment:
missing tests for preselected item  being greyed out in list?

Reply:
Ah, already there in `TokenSelector::test_clickNonInteractiveToken()`